wandb/sdk/artifacts/artifact.py

```diff
@@ -765,9 +766,13 @@ def wait(self, timeout: Optional[int] = None) -> "Artifact":
             timeout: Wait up to this long.
         """
         if self.is_draft():
-            if self._save_future is None:
+            if self._save_handle is None:
                 raise ArtifactNotLoggedError(self, "wait")
-            result = self._save_future.get(timeout)
+
+            if timeout is None:
+                timeout = -1
+            termlog(f"Waiting for artifact {self.name} to be commited...")
```

Comment:
Nit: spelling, same on line 785.
```suggestion
            termlog(f"Waiting for artifact {self.name} to be committed...")
```

Reply:
fixed